lib/ansiblelint/constants.py

```diff
@@ -1,7 +1,15 @@
 """Constants used by AnsibleLint."""
 import os.path
+import sys
+
+if sys.version_info >= (3, 8):
+    from typing import Literal  # pylint: disable-msg=E0611
+else:
+    from typing_extensions import Literal
```

Comment:
The proper fallback idiom is
```suggestion

try:
    from typing import Literal  # pylint: disable-msg=E0611
except ImportError:
    from typing_extensions import Literal
```

Reply:
Have you tried it locally? I suspect your approach may upset one of our linters. I am used to the same idiom, and if I remember well, there was an issue that forced me to do it the ugly way.